### ISSUE TITLE
Toggle fullscreen feature

### ISSUE THREAD

**Issue by R4stafa** [xmonad/xmonad-contrib#394]
I use XMonad.Layout.Fullscreen for fullscreen support and the AddFullscreen and RemoveFullscreen messages to manually make windows fullscreen but there is no easy way (at least I can't see one) to toggle the fullscreen state of a window say with a keybinding. So I went ahead and built my own xmonad-contrib where I added a ToggleFullscreen message and it works. I thought this feature should be in the official xmonad-contrib but I wanted to discuss this first before making a PR.

Sorry if I do something the the wrong way I am new to contributing to open source projects.

**Comment by elkowar** [comment]
the whole point of contrib is that it's easy to PR to! So if you already have the code, there is no harm in simply doing the PR, and seeing what happens!

**Comment by R4stafa** [comment]
Ok I created the PR #395

**Comment by slotThe** [comment]
At this point this is basically a duplicate of https://github.com/xmonad/xmonad-contrib/issues/456.  This will be added once https://github.com/xmonad/xmonad-contrib/pull/625 is merged
